### ISSUE TITLE
Revocation notification cannot be processed in 0.2.0-alpha.57

### ISSUE THREAD

**Issue by marcinkiewiczblazej** [hyperledger/aries-framework-javascript#724]
I'm using this stack:
- ACA-Py 0.7.3 as backend agent
- AFJ 0.2.0-alpha.57 as a react native wallet agent

When ACA-Py is sending revocation notification I'm getting error in the mobile app with the following message:
```
"Incorrect revocation notification threadId format: \nindy::5pnRkcshgidE6AoRaLdXce:4:5pnRkcshgidE6AoRaLdXce:3:CL:57051:Agent_Alias.Schema_tag:CL_ACCUM:7c3f4614-9b35-4444-98c3-fd4a9b048310::2\ndoes not match\n\"indy::<revocation_registry_id>::<credential_revocation_id>\"",
```

When creating a schema we're using concatenation of agent alias and the schema name (we replace spaces with underscores). `.` is the character that is used as a separator, e.g. `Agent_Alias.Schema_tag`. ACA-Py allows to use this value as a tag, but the regex used in AFJ `RevocationService.ts` line 64 rejects it. It's rejected because of `.` , is it something that should be updated on the ACA-Py side or in the AFJ?

**Comment by marcinkiewiczblazej** [comment]
If i replace `.` with `__` then it's working fine.

**Comment by TimoGlastra** [comment]
Good catch @marcinkiewiczblazej! I think we should loosen the regex on the AFJ side and allow dots. We actually took this regex from ACA-Py, but it looks like they're not using it themselves maybe?

Are you willing to open a PR for this?

**Comment by marcinkiewiczblazej** [comment]
I'll prepare PR with the fix. I've been digging depper and I've found the regex. It's a bit different in the schema tag part. In ACA-Py there's `(:.+)?` vs  `(?::[\dA-z]+)?`

This is the regex I've found. There's a chance there's a different one in other place ;]
https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/messaging/valid.py#L410
